aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/AWSXRayRecorder.java

```diff
@@ -1082,4 +1083,12 @@ public String currentFormattedId() {
             return null;
         }
     }
+
+    /**
+     * Configures this {@code AWSXRayRecorder} to add valid TraceId in all segments even NoOp ones that usually have
+     * a fixed value.
+     */
+    public void alwaysCreateTraceID(final boolean alwaysCreateTraceId) {
```

Comment:
It would probably be best if this method was explicitly named as a setter, e.g. `setForcedTraceIdGeneration`.